src/test/kotlin/io/github/mojira/arisa/modules/CrashModuleTest.kt

```diff
@@ -30,17 +30,48 @@ const val EXAMPLE_CRASH = "---- Minecraft Crash Report ----\n" +
         "Is Modded: Probably not. Jar signature remains and client brand is untouched.\n"
 
 const val SERVER_UNMODDED_CRASH = "---- Minecraft Crash Report ----\n" +
-        "// Daisy, daisy...\n" +
+        "// Oh - I know what I did wrong!\n" +
         "\n" +
-        "Time: 6/28/17 11:44 AM\n" +
-        "Description: Initializing game\n" +
+        "Time: 6/5/18 9:20 PM\n" +
+        "Description: Exception generating new chunk\n" +
+        "\n" +
+        "java.util.concurrent.ExecutionException: java.lang.RuntimeException: We are asking a region for a chunk out of bound | -174 8\n" +
         "\n" +
-        "org.lwjgl.LWJGLException: Pixel format not accelerated\n" +
         "-- System Details --\n" +
         "Details:\n" +
-        "\tMinecraft Version: 1.8.9\n" +
-        "\tJava Version: 1.8.0_131, Oracle Corporation\n" +
-        "Is Modded: Unknown\n"
+        "\tMinecraft Version: 1.13-pre1\n" +
+        "\tOperating System: Linux (amd64) version 4.4.0-98-generic\n" +
+        "\tJava Version: 1.8.0_151, Oracle Corporation\n" +
+        "\tJava VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation\n" +
+        "\tMemory: 614132352 bytes (585 MB) / 988282880 bytes (942 MB) up to 3340763136 bytes (3186 MB)\n" +
+        "\tJVM Flags: 2 total; -Xmx3584M -XX:MaxPermSize=256M\n" +
+        "\tProfiler Position: N/A (disabled)\n" +
+        "\tPlayer Count: 1 / 20; [so['CENSORED'/351, l='Vanilla', x=8.50, y=72.00, z=121.50]]\n" +
+        "\tData Packs: vanilla\n" +
+        "\tIs Modded: Unknown (can't tell)\n" +
+        "\tType: Dedicated Server (map_server.txt)\n"
+
+const val SERVER_MODDED_CRASH = "---- Minecraft Crash Report ----\n" +
+        "// Oh - I know what I did wrong!\n" +
+        "\n" +
+        "Time: 6/5/18 9:20 PM\n" +
+        "Description: Exception generating new chunk\n" +
+        "\n" +
+        "java.util.concurrent.ExecutionException: java.lang.RuntimeException: We are asking a region for a chunk out of bound | -174 8\n" +
+        "\n" +
+        "-- System Details --\n" +
+        "Details:\n" +
+        "\tMinecraft Version: 1.13-pre1\n" +
+        "\tOperating System: Linux (amd64) version 4.4.0-98-generic\n" +
+        "\tJava Version: 1.8.0_151, Oracle Corporation\n" +
+        "\tJava VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation\n" +
+        "\tMemory: 614132352 bytes (585 MB) / 988282880 bytes (942 MB) up to 3340763136 bytes (3186 MB)\n" +
+        "\tJVM Flags: 2 total; -Xmx3584M -XX:MaxPermSize=256M\n" +
+        "\tProfiler Position: N/A (disabled)\n" +
+        "\tPlayer Count: 1 / 20; [so['CENSORED'/351, l='Vanilla', x=8.50, y=72.00, z=121.50]]\n" +
+        "\tData Packs: vanilla\n" +
+        "\tIs Modded: Definitely; Client brand changed to 'fabric'\n" +
```

Comment:
This should probably be <code>Definitely; <ins>Server</ins> brand changed to 'fabric'</code> to cover the dedicated server case.

Reply:
Agreed.

Might also make sense to include one of those Likely, jar signature invalidated ones.
I should be able to dig one of those up tomorrow.

Reply:
Done. Waiting for the signature invalidated ones

Reply:
Server:
```
---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 04.10.19 17:39
Description: Exception ticking world

java.lang.NoSuchFieldError: DO_DAYLIGHT_CYCLE

-- System Details --
Details:
	Minecraft Version: 1.14.4
	Minecraft Version ID: 1.14.4
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1192941208 bytes (1137 MB) / 1845493760 bytes (1760 MB) up to 2147483648 bytes (2048 MB)
	CPUs: 4
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Player Count: 0 / 8; []
	Data Packs: vanilla
	Type: Integrated Server (map_client.txt)
	Is Modded: Very likely; Jar signature invalidated
```
Client:
```
---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 06.10.19 22:01
Description: Unexpected error

net.minecraft.text.TranslationException: Error while parsing: TranslatableComponent{key='%s updates in tick %2: %s block, %s redstone, %s qc', args=[15031, 4336481, 0, 3347, 11684], siblings=[], style=Style{hasParent=false, color=null, bold=null, italic=null, underlined=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null}}

-- System Details --
Details:
	Minecraft Version: 1.14.4
	Minecraft Version ID: 1.14.4
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_221, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 440810624 bytes (420 MB) / 1142423552 bytes (1089 MB) up to 1890582528 bytes (1803 MB)
	CPUs: 4
	JVM Flags: 0 total; 
	Launched Version: 1.14.4
	LWJGL: 3.2.2 build 10
	OpenGL: AMD Radeon (TM) R9 380 Series GL version 4.5.13541 Compatibility Profile Context 24.20.13019.1008, ATI Technologies Inc.
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: vanilla, file/1.13 iRed [Default] v0.02c.zip
	Current Language: English (US)
	CPU: <unknown>
```

One of them should be enough, the is modded value looks the same.

Reply:
done